### DISCUSSION TITLE
Old source code

### DISCUSSION THREAD

**Discussion by ghost** [ldtteam/minecolonies#10434] [Q&A]
Please update the source code, because many points in the code are already outdated.

**Comment by Raycoms** [comment]
Wat?

**Comment by ghost** [comment]
The source code is posted on GitHub. Many scripts in it are no longer relevant.

**Comment by ghost** [comment]
an example of legacy code: src/main/java/com/minecolonies/core/colony/buildings and in it the script "AbstractBuildingGuaeds.java"

**Comment by Thodor12** [comment]
That file is not legacy...
